code/modules/client/preferences/species_features/basic.dm

```diff
@@ -98,6 +98,11 @@
 	target.grad_color[GRADIENT_FACIAL_HAIR_KEY] = value
 	target.update_hair()
 
+/datum/preference/color/facial_hair_gradient/is_accessible(datum/preferences/preferences)
+	if (!..(preferences))
```

Comment:
```suggestion
	. = ..()
	if(!.)
```

All args are passed down if none is specified inside the parenthesis of the `..()` function, and since `preferences` is the only argument here this should work just as fine

Reply:
This is the way the rest of prefs menu code is already, it's a personal preference to prefer explicitness over implicitness, and because I really hate `.`.

Reply:
Ok, I see.